libsupport/include/katana/URI.h

```diff
@@ -12,12 +12,6 @@
 namespace katana {
 
 class KATANA_EXPORT Uri {
-  std::string scheme_;
-  std::string path_;
-  std::string string_;
-
-  Uri(std::string scheme, std::string path);
```

Comment:
Appreciate moving this down to an explicit `private` block, but out of curiosity, would these have been public by default?

Reply:
They are private by default.  But our coding standard is to put private data at the bottom of the class.